src/components/_modals/BuyOrSellModal.tsx

```diff
@@ -39,75 +43,82 @@ export const BuyOrSellModal = ({
+                      boxSize={6}
+                    />
+                    <Text fontSize="xl" fontWeight="bold">
+                      {value.name}
+                    </Text>
+                  </HStack>
+                  <ChevronRightIcon />
+                </HStack>
+              </Link>
+            )
+          }
+          return (
+            <Link
+              href={value.url}
+              key={value.name}
+              onClick={() => {
+                analytics.track("strategy.buy-sell", {
+                  strategyCard: cellarDataMap[id].name,
+                  platformSelection: "Uniswap",
+                  landingType: landingType(),
+                })
               }}
+              target="_blank"
+              style={{ textDecoration: "none" }}
             >
-              <HStack spacing={4}>
-                <Image
-                  alt="uniswap icon"
-                  src="/assets/icons/uniswap.png"
-                  boxSize={6}
-                />
-                <Text fontSize="xl" fontWeight="bold">
-                  Uniswap
-                </Text>
+              <HStack
+                justifyContent="space-between"
+                backgroundColor="surface.secondary"
+                padding={4}
+                borderRadius="xl"
+                _hover={{
+                  backgroundColor: "purple.dark",
+                }}
+              >
+                <HStack spacing={4}>
+                  <Image
+                    alt="uniswap icon"
+                    src={value.logo}
+                    boxSize={6}
+                  />
+                  <Text fontSize="xl" fontWeight="bold">
+                    {value.name}
+                  </Text>
+                </HStack>
+                <ExternalLinkIcon />
               </HStack>
-              <ExternalLinkIcon />
-            </HStack>
-          </Link>
-        )}
-        <Link
-          href={`/strategies/${id}/manage`}
-          onClick={() => {
-            analytics.track("strategy.buy-sell", {
-              strategyCard: cellarDataMap[id].name,
-              platformSelection: "Sommelier",
-              landingType: landingType(),
-            })
-          }}
-          style={{ textDecoration: "none" }}
-        >
-          <HStack
-            justifyContent="space-between"
-            backgroundColor="surface.secondary"
-            padding={4}
-            borderRadius="xl"
-            _hover={{
-              backgroundColor: "purple.dark",
-            }}
-          >
-            <HStack spacing={4}>
-              <Image
-                alt="uniswap icon"
-                src="/assets/icons/somm.png"
-                boxSize={6}
-              />
-              <Text fontSize="xl" fontWeight="bold">
-                Sommelier
-              </Text>
-            </HStack>
-            <ChevronRightIcon />
-          </HStack>
-        </Link>
+            </Link>
+          )
+        })}
```

Comment:
It may also be worth abstracting these two views to their own components and importing them into the if/else logic. Not 100% necessary, but worth considering